test/db/cmd/types

```diff
@@ -278,18 +278,108 @@ RUN
 NAME=te
 FILE=-
 CMDS=<<EOF
-td enum pe_machine { IMAGE_FILE_MACHINE_IA64=0x200, IMAGE_FILE_MACHINE_I386=0x14c };
+td "enum pe_machine { IMAGE_FILE_MACHINE_IA64=0x200, IMAGE_FILE_MACHINE_I386=0x14c };"
 te pe_machine 0x14c
 EOF
 EXPECT=<<EOF
 IMAGE_FILE_MACHINE_I386
 EOF
 RUN
 
+NAME=tef
+FILE=-
+CMDS=<<EOF
+td "enum FOO { A=1, B=0x100, C=0xFF, D=4, E=5 };"
+td "enum BAR { G=0x800, Z=0xEE, P=4, E=5 };"
+tef 0x800
+tef 0x100
+tef 4
+tef 0x5
+EOF
+EXPECT=<<EOF
+BAR.G
+FOO.B
+BAR.P
+FOO.D
+BAR.E
+FOO.E
+EOF
+RUN
+
+NAME=tej
+FILE=-
+CMDS=<<EOF
+td "enum FOO { A=1, B=0x100, C=0xFF, D=4, E=5 };"
+td "enum BAR { G=0x800, Z=0xEE, P=4, E=5 };"
+tej FOO
+tej BAR
+tej
```

Comment:
this should output an array, not just 2 objects, I think.

Reply:
@XVilka seen this?

Reply:
It will require passing `PJ` pointer around :(

Reply:
it is necessary, I think. If you have a json command that returns more than one object, they should be wrapped in `[...]`.

Reply:
Done, albeit in an ugly way.